subsys/bluetooth/host/gatt.c

```diff
@@ -3671,6 +3830,20 @@ static int bt_gatt_clear_cf(u8_t id, const bt_addr_le_t *addr)
 	return settings_delete(key);
 #endif /* CONFIG_BT_GATT_CACHING */
 	return 0;
+
+}
+
+static int bt_gatt_clear_sc(u8_t id, const bt_addr_le_t *addr)
```

Comment:
I just tried to follow naming conventions from that file to make it more readable, or easier to change in a future cleanup, as we already have:
static int bt_gatt_clear_ccc(...);
static int bt_gatt_clear_cf(...);

(I thought about renaming the above two functions but I'd rather not add too much to this PR)
Would you still prefer for me to rename it?

Reply:
Yes, I noticed that they other functions where named the same way. Almost all functions in this file name `bt_gatt_*` is non-static. So I would at least rename the ones you added.

It's ok to leave the others, as they are not part of your changes but if you decide to rename them you could do it in a new commit, so that it is easy to separate the changes.

Reply:
Pushed in last commit!